.gitignore

```diff
@@ -148,3 +148,6 @@ metastore_db
 # dependency management
 Pipfile
 Pipfile.lock
+
+# contract testing files
+tests/integration/cloud/rest_contracts/pacts/great_expectations-mercury.json
```

Comment:
should we ignore all JSON files in this directory? Or perhaps the entire directory?

Reply:
Yeah I meant to do the entire directory and forgot ✅